include/SFML/Window/Context.hpp

```diff
@@ -84,6 +84,9 @@ class SFML_WINDOW_API Context : GlResource
     ////////////////////////////////////////////////////////////
     Context& operator=(const Context&) = delete;
 
+    Context(Context&& other) noexcept;
+    Context& operator=(Context&& other) noexcept;
+
```

Comment:
Please match the code style of the rest of this file. There are comment blocks you're supposed to add. See something like Shader.hpp for what to include in those comment blocks.